src/test/java/org/folio/permstest/RestVerticleTest.java

```diff
@@ -87,7 +87,7 @@
   static int port;
 
   @Rule
-  public Timeout rule = Timeout.seconds(10);
+  public Timeout rule = Timeout.seconds(10); // set to 900 to test MODPERMS-185
```

Comment:
```
  @Test
  public void testPermsAssignMigration50(TestContext context) {
    testPermsAssignMigration(context, 50);
  }

  @Ignore("Takes many minutes")
  @Test(timeout = 900000l)
  public void testPermsAssignMigration5000(TestContext context) {
    testPermsAssignMigration(context, 5000);
  }

```

Reply:
Done, but I can't get the per test timeout to really work ..